pop.css

```diff
@@ -0,0 +1,151 @@
+.pop-up-background {
+  height: 100vh;
+  width: 100vw;
+  position: fixed;
+  top: 0;
+  left: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  backdrop-filter: blur(10px);
+}
+
+.pop-up-card {
+  border: 1px solid #dfe1e6;
+  border-radius: 4.26vw;
+  background-color: #fff;
+  width: 91.46vw;
+  overflow: auto;
+}
+
```

Comment:
- [ ] Nice work sofar however you need to add padding on top and bottom your popup window and reduce the size of the image so that it looks like the template one, remeber our design should look exactly like the figma template. 

your design | the template
---|---
<img width="1440" alt="image" src="https://github.com/pulgosodog/Portfolio-template-1/assets/65068771/4753828b-84b4-4254-b611-0d670a2aa814"> | <img width="691" alt="image" src="https://github.com/pulgosodog/Portfolio-template-1/assets/65068771/dba1c301-a026-4657-8f7a-2a2f311fa60c">

Reply:
My apologizes but i think the image is the right size, I'm stricly adhearing to the figma template 1 ratios given for the pop up:
Figma: 
![image](https://github.com/pulgosodog/Portfolio-template-1/assets/115130832/69f7de2f-6d11-4984-a942-94ddb699cf53)
My project:
![image](https://github.com/pulgosodog/Portfolio-template-1/assets/115130832/cac2e673-6a53-4c63-873e-3d222b7f6069)

Reply:
Kindly have a look at the margin (intead of padding (used as separation) on the top and bottom, it matches altmost perfectly with the figma design:
![image](https://github.com/pulgosodog/Portfolio-template-1/assets/115130832/e21105c6-90f3-4302-914c-344067aa9215)
![image](https://github.com/pulgosodog/Portfolio-template-1/assets/115130832/b8be19fe-9614-4df3-bff4-14797a62e619)